### ISSUE TITLE
I would like to know about milestone for LAMMPS kokkos version

### ISSUE THREAD

**Issue by jimmylegendary** [CompPhysVienna/n2p2#160]
**The question**
I would like to know about milestone for LAMMPS kokkos version
even though it is vague

**What did you already try?**
I have read issues in the below
https://github.com/CompPhysVienna/n2p2/pull/88
https://github.com/ECP-copa/CabanaMD/issues/97

**Background**
What system are you working on?
Linux

**Comment by dreith** [comment]
Indeed, a kokkos version of the LAMMPS n2p2 plugin would be really helpful.
